### PR TITLE
Add Bitbucket

### PR DESCRIPTION
**Pull request by yudori**
Adds bitbucket for source control

### CODE REVIEW

File: data/tools/bitbucket.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "slug": "bitbucket",
+    "name": "bitbucket",
+    "description": "Bitbucket is a web-based hosting service that supports both Mecurial and Git version control systems. It offers commercial and free accounts. It supports collaboration with unlimited private repos, diff views, CI build integrations, large file storage, pipelines, JIRA and other 3rd party integrations.",
+    "tags": [    
+      "commercial",
+      "vcs",
+      "git",
+      "mecurial"
```
Comment (rdsubhas) [inline data/tools/bitbucket.json]: Hi, thanks for the contrib, should this be `mercurial`?

Comment (yudori) [inline data/tools/bitbucket.json]: Oops!...my bad :)

**Comment by yudori** [comment]
Changed!

**Comment by rdsubhas** [comment]
Thanks for the contrib, merging!

**Pull request merged**